shark/examples/eager_mode.py

```diff
@@ -0,0 +1,123 @@
+import torch
```

Comment:
do we want any copyright/licensing informatiom here and in all the other new files? Probably a question for @powderluv